README.md

```diff
@@ -454,7 +454,68 @@ public class PersonRepositoryIntegrationTest {
 Você pode ver que nosso teste de integração segue a mesma estrutura *Arrange, Act, Assert* dos testes de unidade. Eu lhe disse que este era um conceito universal!
 
 
-### <a id="sec-integration-separated-services"></a>Integration With Separate Services
+### <a id="sec-integration-separated-services"></a>Integração com serviços separados
+
+Nosso microsserviço se comunica com o [darksky.net](https://darksky.net/), uma API REST de meteorologia. É claro que queremos garantir que nosso serviço mande requisições e analise as respostas corretamente.
+
+Queremos evitar acessar os servidores reais do darksky ao executar testes automatizados. Os limites de cota do nosso plano gratuito são apenas parte do motivo. A verdadeira razão é desacoplamento. Nossos testes devem ser executados independentemente do que as pessoas amáveis do [darksky.net](https://darksky.net/) estejam fazendo. Mesmo quando sua máquina não consiga acessar os servidores da darksky ou os servidores da darksky estejam fora do ar para manutenção.
```

Comment:
Favor colocar a palavra darksky em itálico. Não apenas aqui, mas nos demais lugares onde ela aparece.